packages/@okta/vuepress-site/docs/guides/token-inline-hook/parse-hook/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Parse the Token Inline Hook request
+---
+
+The external service in this scenario requires code to handle the Token Inline Hook request from the Okta org. Use the Glitch example to either build or copy the code (re-mix on Glitch) that parses the Token Inline Hook call.
```

Comment:
It doesn't hurt to hyperlink "Glitch example" here again. I did re-mix it in the earlier step but nice to hyperlink it again to confirm that I've re-mixed the right one.